src/helpers/index.tsx

```diff
@@ -205,3 +210,11 @@ export const subtractDates = (dateA: Date, dateB: Date) => {
     seconds,
   };
 };
+
```

Comment:
Maybe give these two functions consistent names? i.e. `toBN`\`fromBN` or `bnToNum`\'numToBn`
Not a big thing but would be easier to remember when trying to call then

Reply:
I will remove these two functions as they actually aren't being used.